### PR TITLE
Add "Legal" justification of the collection process - Update Design - Translation

### PR DESCRIPTION
**Pull request by fe51**
This PR introduces the following changes :

- Addition of a drop-down section detailing our data collection approach, and with the valuable assistance of a lawyer aligning us with good data collection practice (NO personal data involved here)
- Update of the application flow:
      - Upload section completely moved in the central page instead of the sidebar. 
      - Add Pyronear description in sidebar
- Translation of the page into French as we mainly target a French-speaking audience

Also, I was unable to simply properly display the pyronear "logo_letters" from an url. So, for now, I added the logo to the repo.

You can see how it renders with the screenshots below : 
image

image

Happy to discuss it, especially the wording :)

### CODE REVIEW

**Comment by fe51** [comment]
Hi @frgfm ! Can you have a look at the PR, especially the wording :)

File: src/app.py [hunk 1]
```diff
@@ -19,50 +20,70 @@ def main():
...
     # Designing the interface
-    st.title("Wildfire image crowdsourcing")
-    st.write("The image you upload here will be used to create a public image dataset for wildfire detection.")
+    #st.title("Collecte de photos de feux de forêt")
+    st.title("Améliorez la detection des départs de feux")
+    st.write("La photo que vous partagez ici sera utilisée pour créer un jeu de données public "
+             "pour la detection de feux de forêts.")
+    st.write("Idéalement partagez nous une photo depuis une vue assez dégagée sur une zone forestière, avec et/ou sans flammes/fumées")
+    #st.write("Lorsque vous partagez une photo de départ de feux, n'hésitez pas à "
+    #        "également partager une photo du même point de vue sans fumées ou flammes.")
```
Comment (frgfm) [inline src/app.py:29]: Are the commented sections here on purpose?

File: src/app.py [hunk 2]
```diff
@@ -19,50 +20,70 @@ def main():
...
+
     # Set the columns
     cols = st.columns((2, 1))
-    cols[0].subheader("Input image")
-    cols[1].subheader("Image tags")
+    cols[0].subheader("Selection de la photo")
```
Comment (frgfm) [inline src/app.py:33]: Missing the accent on sélection

**Review by frgfm — COMMENTED** [review]
I think the two open PRs are interconnected (the second one was most likely started from the branch of this PR :sweat_smile: )

File: src/app.py (hunk 1 above)
Comment (fe51) [reply src/app.py:29]: Well, which wording do you prefer ? the one in comments or the one in line 27 ? I will delete comments/l27 then

File: src/app.py (hunk 2 above)
Comment (fe51) [reply src/app.py:33]: Nice catch

File: src/app.py [hunk 3]
```diff
@@ -19,50 +20,70 @@ def main():
...
-    st.write("The image you upload here will be used to create a public image dataset for wildfire detection.")
+    #st.title("Collecte de photos de feux de forêt")
+    st.title("Améliorez la detection des départs de feux")
+    st.write("La photo que vous partagez ici sera utilisée pour créer un jeu de données public "
+             "pour la detection de feux de forêts.")
+    st.write("Idéalement partagez nous une photo depuis une vue assez dégagée sur une zone forestière,"
+             " avec et/ou sans flammes/fumées")
+    #st.write("Lorsque vous partagez une photo de départ de feux, n'hésitez pas à "
+    #        "également partager une photo du même point de vue sans fumées ou flammes.")
+
```
Comment (frgfm) [inline src/app.py:31]: Let's remove the commented sections, no?

**Review by frgfm — APPROVED** [review]
Apart from the commented lines to remove, it looks good to me :+1:

File: src/app.py [hunk 4]
```diff
@@ -19,50 +20,68 @@ def main():
     )
 
     # Designing the interface
-    st.title("Wildfire image crowdsourcing")
-    st.write("The image you upload here will be used to create a public image dataset for wildfire detection.")
+    #st.title("Collecte de photos de feux de forêt")
```
Comment (frgfm) [inline src/app.py:23]: There is still one over here :upside_down_face:

**Review by frgfm — APPROVED** [review]
You left one out :sweat_smile:

**Review by frgfm — APPROVED** [review]
All good!

**Pull request merged**